### ISSUE TITLE
Compatible with Tailwindcss 2

### ISSUE THREAD

**Issue by molfar** [theMosaad/tailwindcss-capsize#5]
I've encouraged with error Error: Cannot find module './lib/util/createPlugin' - code: 'MODULE_NOT_FOUND'
With Tailwindcss 2.0.2

**Comment by theMosaad** [comment]
Can you send your setup to reproduce the issue?
I'm using `"@themosaad/tailwindcss-capsize": "^0.5.0",` with `"tailwindcss": "npm:@tailwindcss/postcss7-compat@^2.0.2"` at the moment with no problems.

**Comment by molfar** [comment]
I have the same package.json
{
  "dependencies": {
    "@tailwindcss/postcss7-compat": "^2.0.2",
    "@themosaad/tailwindcss-capsize": "^0.5.0",
    "autoprefixer": "^9.8.6",
    "postcss": "^7.0.35",
    "postcss-cli": "^7.1.2",
    "tailwindcss": "npm:@tailwindcss/postcss7-compat@^2.0.2",
    "tailwindcss-textshadow": "^2.1.3"
  },
  "scripts": {
    "dev": "postcss src/*.css --dir public/css --verbose --watch"
  }
}

**Comment by theMosaad** [comment]
It might have to do with tailwind.config.js, can you send it over?

**Comment by molfar** [comment]
```
const colors = require('tailwindcss/colors')
const defaultTheme = require('tailwindcss/defaultTheme')

module.exports = {
  purge: [],
  darkMode: false, // or 'media' or 'class'
  theme: {
    screens: defaultTheme.screens['xs'],
    capsize: {
      fontMetrics: {
        'open': {
          capHeight: 1434,
          ascent: 2115,
          descent: -270,
          lineGap: 0,
          unitsPerEm: 2048
        }
      },
    },
    extend: {
      fontFamily: {
        'open': [
          '"Open 24 Display St"',
          ...defaultTheme.fontFamily.mono,
        ]
      },
    },
  },
  variants: {
    extend: {},
  },
  plugins: [
    require('tailwindcss-textshadow'),
    require('@themosaad/tailwindcss-capsize')
  ],
}

```

**Comment by theMosaad** [comment]
I tested with a fresh nextjs installation and the only warning I got was when installing `@themosaad/tailwindcss-capsize@0.5.0" has incorrect peer dependency "tailwindcss@^1.5.0"`
However, the build completed and it trimmed the text.

Can you share more of your setup so I can reproduce? You can do so privately if you wish on twitter [@theMosaad](https://twitter.com/theMosaad)

**Comment by molfar** [comment]
@theMosaad you know, I cant reproduce error right now. I created a blank project with the same dependencies, and there is no error there. Moreover, I returned the tailwindcss-capsize module to my real project, and I can catch an error here too. I've saved full backtrace, I encouraged before (attaching full message just for history). But as there's no error now, I'm closing the issue. Thankyou for responses and for such useful integration for tailwindcss.

```
Error: Cannot find module './lib/util/createPlugin'
Require stack:
- /Users/molfar/Desktop/project_name/node_modules/@themosaad/tailwindcss-capsize/node_modules/tailwindcss/plugin.js
- /Users/molfar/Desktop/project_name/node_modules/@themosaad/tailwindcss-capsize/src/index.js
- /Users/molfar/Desktop/project_name/tailwind.config.js
- /Users/molfar/Desktop/project_name/node_modules/tailwindcss/lib/index.js
- /Users/molfar/Desktop/project_name/node_modules/import-from/index.js
- /Users/molfar/Desktop/project_name/node_modules/import-cwd/index.js
- /Users/molfar/Desktop/project_name/node_modules/postcss-load-config/src/options.js
- /Users/molfar/Desktop/project_name/node_modules/postcss-load-config/src/index.js
- /Users/molfar/Desktop/project_name/node_modules/postcss-cli/index.js
- /Users/molfar/Desktop/project_name/node_modules/postcss-cli/bin/postcss
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:919:15)
    at Function.Module._load (node:internal/modules/cjs/loader:763:27)
    at Module.require (node:internal/modules/cjs/loader:991:19)
    at require (node:internal/modules/cjs/helpers:92:18)
    at Object.<anonymous> (/Users/molfar/Desktop/project_name/node_modules/@themosaad/tailwindcss-capsize/node_modules/tailwindcss/plugin.js:1:22)
    at Module._compile (node:internal/modules/cjs/loader:1102:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1131:10)
    at Module.load (node:internal/modules/cjs/loader:967:32)
    at Function.Module._load (node:internal/modules/cjs/loader:807:14)
    at Module.require (node:internal/modules/cjs/loader:991:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/molfar/Desktop/project_name/node_modules/@themosaad/tailwindcss-capsize/node_modules/tailwindcss/plugin.js',
    '/Users/molfar/Desktop/project_name/node_modules/@themosaad/tailwindcss-capsize/src/index.js',
    '/Users/molfar/Desktop/project_name/tailwind.config.js',
    '/Users/molfar/Desktop/project_name/node_modules/tailwindcss/lib/index.js',
    '/Users/molfar/Desktop/project_name/node_modules/import-from/index.js',
    '/Users/molfar/Desktop/project_name/node_modules/import-cwd/index.js',
    '/Users/molfar/Desktop/project_name/node_modules/postcss-load-config/src/options.js',
    '/Users/molfar/Desktop/project_name/node_modules/postcss-load-config/src/index.js',
    '/Users/molfar/Desktop/project_name/node_modules/postcss-cli/index.js',
    '/Users/molfar/Desktop/project_name/node_modules/postcss-cli/bin/postcss'
  ]
}
```

**Comment by theMosaad** [comment]
@molfar, turns out you were right! I had tailwindcss v1 as a dependency instead of a Devdependency. Fixed it in v1.0.0. Not sure how it went unnoticed that long so appreciate your feedback.

please, install the latest version of `@themosaad/tailwindcss-capsize` it and let me know if it fixed the issue for you.

**Comment by molfar** [comment]
Confirm, now it works as expected! **No error** neither in blank project nor in existing.
